programs/programs/policyengine/calculators/dependencies/member.py

```diff
@@ -109,11 +109,25 @@ def value(self):
         return int(self.member.calc_gross_income('yearly', ['unearned']))
 
 
-class SsiDisabledDependency(Member):
-    field = "is_ssi_disabled"
+class IsDisabledDependency(Member):
+    field = "is_disabled"
 
     def value(self):
-        return self.member.has_disability()
+        return self.member.disabled or self.member.long_term_disability
+
+
+class IsBlindDependency(Member):
+    field = 'is_blind'
+
+    def value(self):
+        return self.member.visually_impaired
+
+
+class SsiReportedDependency(Member):
+    field = 'ssi_reported'
+
+    def value(self):
```

Comment:
This seems like a placeholder or stub implementation? It might be helpful to include a comment explaining the reason for always returning 0.

Reply:
Updated